README.md

```diff
@@ -52,12 +53,9 @@ curl --request POST \
 }'
 ```
 
-- refresh
+- Refresh token
 ```
 curl --request POST \
   --url http://localhost:3000/refresh \
-  --header 'content-type: application/json' \
-  --data '{
-	"token": "jwt-token"
-}'
+  --header 'authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY0YWM2ODMzLWFhYTItNDA3MS1hY2RmLTMxMDRiODg4NDdlMiIsImVtYWlsIjoidXNlckBleGFtcGxlLmNvbSIsImV4cCI6MTU2MzMxMzg2NH0.Oc1TxVQoxmLcT5TFwxdE2GEKjKpM22hlu3jpAuBNoUQ'
```

Comment:
Please, replace JWT token with some placeholder, like `[JWT_TOKEN]`

Reply:
why? it's just an example..

Reply:
fixed